packages/framework/src/Migrations/DataModifiers/CountryDataModifierVersion20190121094400.php

```diff
@@ -0,0 +1,192 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Migrations\DataModifiers;
+
+use function array_diff;
```

Comment:
It's optimization for parser. With native functions, you should use either `\array_diff(` or import the function as it is here. 
By default parser looking for the function in a namespace and after that in a global scope. With import or FQN you force the parser to look in a global scope first. 
It's considered a good practice.

And because I don't like backward slashes in a front of things in code, I choose this way.

Reply:
Ok, thanks for the explanation. I would love if you could create user story/issue or at least discuss it on our internal meeting so we decide if we want to use it. 

It would be also nice to know some performance impact. 

Until then please remove this code.